ee/modules/502-delivery/templates/argocd/argocd-cm-configmap.yaml

```diff
@@ -13,4 +13,5 @@ data:
     issuer: https://{{ include "helm_lib_module_public_domain" (list . "dex") }}/
     clientID: dex-client-argocd@d8-{{ .Chart.Name }}
     clientSecret: $dex-client-argocd:clientSecret
+  oidc.tls.insecure.skip.verify: "true"
```

Comment:
I guess it works fine if the certificate is not a custom one. Should this line be wrapped with `if` condition?

Reply:
@nabokihms please take a look